src/libsync/account.cpp

```diff
@@ -84,7 +84,9 @@ Account::~Account() = default;
 
 QString Account::davPath() const
 {
-    return QLatin1String("/remote.php/dav/files/") + davUser() + QLatin1Char('/');
+    const QString davPathBase("/remote.php/dav/files/");
+    const QString davPathFinal = !davUser().isEmpty() ? davPathBase + davUser() + QLatin1Char('/') : davPathBase;
```

Comment:
this will generate wrong path in case `davUser()` is empty (OK I concede that the existing code will also fail but it has one less conditional)
why is this change needed ?

Reply:
@mgallien Well, this allows us to always have a proper format for the path. At the moment, we are getting `/remote.php/dav/files//` in case `davUser()`, which is incorrect. It has double `/`.

Reply:
I miss the advantage of returning an invalid URL that is the value of `davPathBase`
why do you want to return it since sync will not work with it ?